tgui/packages/tgui/interfaces/PreferencesMenu/jobs/jobs/janitor.ts

```diff
@@ -6,6 +6,7 @@ const Janitor: Job = {
   description: "Clean up trash and blood. Replace broken lights. \
     Slip people over.",
   department: Service,
+  alt_titles: ["Janitor", "Custodian", "Custodial Technicial", "Sanitation Technician"],
```

Comment:
```suggestion
  alt_titles: ["Janitor", "Custodian", "Custodial Technicial", "Sanitation Technician", "Maid"],
```

its a big 
![image](https://user-images.githubusercontent.com/58394696/134925549-bcfdd9c0-7667-4ff4-823b-cf5eeb07c632.png)
but it gets folk playing janny

Reply:
Eh. Maids are different from Janitors, it makes it look a lot less industrial and more to a personal level, which... I don't know, feels weird to me. Not like if we had troubles finding people to play janitor anyway.